src/main/java/com/revature/services/BatchLogicImpl.java

```diff
@@ -52,4 +56,12 @@ public void updateBatch(Batch batchName) {
 		
 	}
 
+	@Override
+	public Page<Person> getAllPeopleByBatchId(Pageable pageable, Integer id) {
+		 Batch batch = dao.findOne(id);
+		 List<Person> personList = batch.getPersons();
+		Page<Person> personPage = new PageImpl<Person>(personList, pageable, personList.size());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS2293)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "personPage". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1488)